x-pack/plugins/secops/server/graphql/kpi_network/schema.gql.ts

```diff
@@ -11,7 +11,8 @@ export const kpiNetworkSchema = gql`
     networkEvents: Float
     uniqueFlowId: Float
     activeAgents: Float
-    uniquePrivateIps: Float
+    uniqueSourcePrivateIps: Float
```

Comment:
I'm curious which of these actually needs to potentially be null. If we're querying for an ECS field (`destination.ip` in the `unique_private_ips` agg for example), then we know the field exists and we'll always at least get back a number in the response (and could set the default value to `0` in the `getOr` in the adapter). Either way, I'm wondering if we really need to keep the `null`/`empty value` value separate from a `0` for these values as they're metrics. Is it beneficial to the user to see `--` vs `0` for a high level KPI like this in the UI? Does it make sense for it to be actionable for them to reach out to a data engineer/support when they see `--`?

No need for any changes, more-so for discussion. @FrankHassanabad has been good about pointing out intentional vs unintentional `null`'s recently, and this seems like it could be one of those cases as these `null`'s can find their way all the way to the `CardTitle` component.

Reply:
For me it sense separating null / empty from numbers, so user can tell whether it displays correct numbers. `--` is like a hint says `The data here is not shown`, so I think it would be more accurate to describe the situation of the field then showing a `0`. I have seen some tooltips appear on the bottom-right corner when data is not retrieved correctly, probably that's the thing we can show under this case as well?

As for intentional vs unintentional null, I would categorise it to unintentional null. Although we have a fallback value, `--` in UI, but we still want it to be a number here.